### PR TITLE
CSV to tfrecord 

### PR DESCRIPTION
**Pull request by isykasle**
Add support for converting csv dataframes to tfrecord.

### CODE REVIEW

File: cookiepy/generate_tfrecords.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+
+import os
+import io
+import pandas as pd
+import tensorflow as tf
+from .od_methods import DuInterface as du_interface
```
Comment (SpiralOutDotEu) [inline cookiepy/generate_tfrecords.py]: What does Du means?

File: cookiepy/generate_tfrecords.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+from .od_methods import LmInterface as lm_interface
```
Comment (SpiralOutDotEu) [inline cookiepy/generate_tfrecords.py]: What does Lm means?

File: cookiepy/generate_tfrecords.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    grouped = split(examples, "filename")
+    for group in grouped:
+        tf_example = create_tf_example(group, path, label_map, du_interface)
+        writer.write(tf_example.SerializeToString())
+    
+    writer.close()
+    
+    print("Successfully created the TFRecords: {}".format(output_path))
+
+
```
Comment (SpiralOutDotEu) [inline cookiepy/generate_tfrecords.py]: Why so many blank lines? 
**Use PEP 8** and provide clean code

File: cookiepy/generate_tfrecords.py [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+from __future__ import print_function
+from __future__ import absolute_import
+
+import os
+import io
+import pandas as pd
+import tensorflow as tf
+from .od_methods import DuInterface as du_interface
+from .od_methods import LmInterface as lm_interface
+from PIL import Image
```
[the remark this answers is not in the document]
Comment (isykasle) [reply cookiepy/generate_tfrecords.py]: Du means dataset_util.
Lm means label_map_util.
Both are modules from object_detection framework.

File: setup.py [hunk 5]
```diff
@@ -14,7 +14,10 @@
 requirements = [
     'Click>=7.0',
     'numpy',
-    'pandas'
+    'pandas',
+    'pygit2',
```
[the remark this answers is not in the document]
Comment (isykasle) [reply setup.py]: No,so it is deleted now!

File: cookiepy/generate_tfrecords.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (SpiralOutDotEu) [reply cookiepy/generate_tfrecords.py]: I think that giving more clear variable names will be better. Try to change the names of variables.

[the remark this answers is not in the document]
Comment (SpiralOutDotEu) [reply cookiepy/generate_tfrecords.py]: Great! Thanks.

**Pull request merged**